en/flight_controller/cuav_v5_nano.md

```diff
@@ -140,61 +140,44 @@ The *V5 nano* has no over current protection.
 Any multicopter / airplane / rover or boat that can be controlled with normal RC servos or Futaba S-Bus servos.
 The complete set of supported configurations can be seen in the [Airframes Reference](../airframes/airframe_reference.md).
 
+## Compatibility 
 
-## Known Issues
-
-The issues below refer to the *batch number* in which they first appear. 
-The batch number is the last two digits of the *serial number* that appears on a sticker on the side of the flight controller. 
-For example, the serial number V031907XXXX**01** indicates the flight controller was in batch 01.
-
-
-#### GPS not compatible with other devices (Critical) - Batch 01 {#issue_gps_compatible}
-
-The *Neo v2.0 GPS* recommended for use with *CUAV V5+* and *CUAV V5 nano* is not fully compatible with other Pixhawk flight controllers (specifically, the buzzer part is not compatible and there may be issues with the safety switch).
-
-The GPS will not work with other flight controllers, and is the only GPS unit that can be used with the *CUAV V5+* and *CUAV V5 nano*.
-<!-- Nano/80/V5_NANO_RC01_2019_05_29 -->
-
-- *Found:* Batch 01
-- *Fixed:* -
+CUAV adopts some differentiated designs and is incompatible with some hardware, which will be described below.
 
-#### PM2 cannot power the flight controller {#issue_pm2}
+#### Neo v2.0 GPS not compatible with other devices {#compatibility_gps}
 
-`PM2` can only measure battery voltage and current, but **not** power the flight controller.
-
-> **Warning** We recommend you do not connect a power module to this connector when using PX4.
+The *Neo v2.0 GPS* that is recommended for use with *CUAV V5+* and *CUAV V5 nano* is not fully compatible with other Pixhawk flight controllers (specifically, the buzzer part is not compatible and there may be issues with the safety switch).
 
-- *Found:* Batch 01
-- *Fixed:* -
+UAVCAN GPS is ideal for avoiding this compatibility.
```

Comment:
Are you saying they should use a different GPS than the one that is supplied? 
Do we have a page for [this here](https://docs.px4.io/master/en/gps_compass/)? If not, can I have a purchase link?

Reply:
@cuhome ^^^

Reply:
https://store.cuav.net/index.php?id_product=111&rewrite=cuav-neo-v2-pro-can-gps&controller=product

Reply:
http://doc.cuav.net/gps/neo-v2-pro/en/

Reply:
It is recommended that you use a link about uavcan GPS; not just a link to CUAV uavcan gps.

Reply:
I modified this. to say

>  "The UAVCAN [NEO V2 PRO GNSS receiver](http://doc.cuav.net/gps/neo-v2-pro/en/#enable) can also be used, and is compatible with other flight controllers."